src/transformers/models/bit/modeling_bit.py

```diff
@@ -777,6 +777,8 @@ def forward(
     """
 )
 class BitBackbone(BitPreTrainedModel, BackboneMixin):
+    has_attentions = False
```

Comment:
required for conv-based backbones to filter out `output_attentions` kwarg on call